keps/sig-cli/3104-introduce-kuberc/README.md

```diff
@@ -331,6 +337,31 @@ intended behavior and realizing that targeting options effectively addresses the
 use cases. During command execution, a merge will be occur, with inline overrides
 taking precedence over the defaults.
 
+`credentialPluginAllowlist` allows the end-user to provide an array of objects
+describing required conditions for executing a credential plugin binary. The
+overall result of a check against the allowlist will be the logical OR of the
+individual checks against the allowlist's entries. Each allowlist entry MUST
+have at least one nonempty field describing the conditions required for the
+plugin's execution. If multiple fields are specified within an entry, the
+binary in question must meet all of the required conditions in that entry in
+order to be executed. At the outset, the entry object will have only one field,
+`name`. The path of the binary specified in the kubeconfig will be compared
+against that named in the `name` field. This field may contain a basename, or
+the full path of a plugin. To ensure an exact match, `os.LookPath` will be
+called on both the `name` field and the binary named in the `kubeconfig`. The
+resulting absolute paths must match. That is to say, each element in the
+allowlist is a set of criteria; if the binary in question meets all of the
+criteria in at least one **set** of criteria, the plugin will be allowed to
+execute.  If no criteria set succeeds after comparing the binary to all sets of
+criteria, the operation will be immediately aborted and an error returned. If
+`credentialPluginAllowlist` is not provided, or is explicitly made `nil`, all
+binaries will be allowed. If `credentialPluginAllowlist`'s value is set to the
```

Comment:
I think, we should mention about this list will not be applied for some commands such as `kubectl config view` and the reason?.